### PR TITLE
Darren Carrillo: Sprint-Challenge-React-Wars

### PR DESCRIPTION
**Pull request by darrenjcarrillo**

### CODE REVIEW

File: starwars/src/App.js [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
+  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
+  const [people, setPeople] = useState([]);
+
+  useEffect(() => {
+    Axios.get("https://swapi.co/api/people/")
+      .then(response => {
+        const people = response.data.results;
+        console.log("the photo", people);
```
Comment (MrT3313) [inline starwars/src/App.js:28]: any change you copied this from the Nasa project? Just remember to look over all the code of you are copying from YOUR OWN code elsewhere. If this is straight up copied from somebody elses project and therefore you dont truly know whats going on, that would be cheating

File: starwars/src/App.js [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
+      })
+      .catch(error => {
+        console.log("The data was not returned", error);
+      });
+  }, []);
 
   return (
     <div className="App">
       <h1 className="Header">React Wars</h1>
+      <CardContainer>
```
Comment (MrT3313) [inline starwars/src/App.js:39]: Remember your singular and purals. is this holding ONE card or MANY cards?

Comment (MrT3313) [reply starwars/src/App.js:39]: I would have simply passed the data from the API call into a `<CardsContainer>` which would act as a bridge between the App and the indvidiual `StarWarsCard`

File: starwars/src/App.js [hunk 3]
```diff
@@ -1,19 +1,56 @@
...
+        {people.map(item => {
+          return (
+            <StarWarsCard
+              key={item}
```
Comment (MrT3313) [inline starwars/src/App.js:43]: if you want to pass the `key` you need to have a `key` / `index` in the .map() arguments --> .map(item, index)

Comment (MrT3313) [reply starwars/src/App.js:43]: Then key={index}

File: starwars/src/components/StarWarsCard.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+`;
+
+const SpaceButton = styled.button`
+  width: 10rem;
+  height: 3rem;
+  margin: 2rem auto;
+  border: solid 1px grey;
+  font-weight: 600;
+  cursor: pointer;
+  background: ${props => (props.primary ? "#7C6658" : "#fff")};
```
Comment (MrT3313) [inline starwars/src/components/StarWarsCard.js:25]: GREAT!! Passing props / themes into styled components can be tricky!

**Review by MrT3313 — COMMENTED** [review]
- good use of getting your data at the top of the app
-

File: starwars/src/App.js (hunk 1 above)
Comment (darrenjcarrillo) [reply starwars/src/App.js:28]: Yes I copied the axios part on my nasa project just to be a little faster. Won't be doing that anymore. Thank you for pointing it out.

**Pull request merged**